ui/tests/acceptance/pki/pki-engine-workflow-test.js

```diff
@@ -343,6 +343,17 @@ module('Acceptance | pki workflow', function (hooks) {
       await runCommands([`write ${this.mountPath}/root/generate/internal common_name="Hashicorp Test"`]);
       await logout.visit();
     });
+    test('lists the correct issuer metadata info', async function (assert) {
```

Comment:
Would love to see another test that covers the case when the lister has access to read some issuers but not all!